sda-commons-server-jackson/src/test/java/org/sdase/commons/server/jackson/test/JacksonConfigurationTestApp.java

```diff
@@ -195,4 +196,22 @@ public Response createRequiredQueryValidationResource(
       @QueryParam("q") @Valid @NotEmpty String searchString) {
     return Response.ok(searchString).build();
   }
+
+  @POST
+  @Path("/subtypes")
+  @Consumes(MediaType.APPLICATION_JSON)
+  public Response createSubType(@Valid @NotNull AbstractResource requestBody) {
+    return Response.created(URI.create("/subtypes/" + requestBody.getType())).build();
+  }
+
+  @POST
+  @Path("/subtypesTolerant")
+  @Consumes(MediaType.APPLICATION_JSON)
+  public Response createSubTypeAllowNull(@Valid AbstractResource requestBody) {
+    if (requestBody == null) {
```

Comment:
I find that a bit speculative to assume that the reasons was that the subtype is not supported. It's unfortunate, that we can't show a proper error message, e.g. `subtype 'three' is not invalid`. We should probably combine `@Valid` and `@NotNull` here, shouldn't we?